lib/units/device/plugins/cleanup.js

```diff
@@ -60,12 +60,48 @@ module.exports = syrup.serial()
           log.info('Cleanup Bluetooth bonds')
           return service.cleanBluetoothBonds()
         }
+        plugin.cleanTmpFolder = function() {
+          if (!options.cleanupTmpFolder) {
+            log.debug('Skipping tmp folder cleanup phase')
+            return
+          }
+          log.info('Cleanup /data/local/tmp folder')
+          return adb.readdir(options.serial, '/data/local/tmp')
+            .then(function(files) {
+              return Promise.map(files, function(file) {
+                if (file.name === '.' || file.name === '..') {
+                  return
+                }
+                var preserveFiles = [
+                  'minicap.apk',
+                  'minicap',
+                  'minicap.so',
+                  'minitouch',
+                  'minirev'
+                ]
+                if (preserveFiles.indexOf(file.name) !== -1) {
+                  log.debug('Skipping /data/local/tmp/%s', file.name)
+                  return
+                }
+                log.info('Removing /data/local/tmp/%s', file.name)
+                return adb
+                  .shell(options.serial, 'rm -rf /data/local/tmp/' + file.name)
```

Comment:
How will it behave if there is a space in the file name?
For example in case of `a b` won't the argument become `rm -rf /data/local/tmp/a b` so no files will be removed?
The whole file path passed to rm should be quoted perhaps `rm -rf '/data/local/tmp/a b'`.

Reply:
good point. Agree, need to add quotes. Do you know if something else is needed for cleanup
phase ?

Reply:
We had need to cleanup several folders in practise, so decided to make this a bit more generic -> configuration can contains multiple paths that is cleaned up. Will update PR description about feature.